app/src/main/kotlin/com/intive/picover/images/view/ImagesScreen.kt

```diff
@@ -47,11 +43,11 @@ private fun ValidateFirebaseStorageState(state: ImagesStorageState) {
 			)
 		}
 
-		is Error -> {
+		state.isError() -> {
 			// TODO-KMA Handle the error
 		}
 
-		is Loading -> PicoverLoader(modifier = Modifier.fillMaxSize())
+		state.isLoading() -> PicoverLoader(modifier = Modifier.fillMaxSize())
```

Comment:
Probably it could be in new line for a better option for debugging or if it works correctly NVM 😅

Reply:
NVM :D